### ISSUE TITLE
False-positive unresolved-var when using potemkin

### ISSUE THREAD

**Issue by ericdallo** [clj-kondo/clj-kondo#1370]
**version**

`2021.08.07-20210822.045523-16`

**platform**
 
JVM
 
**editor**

Happens with CLI and clojure-lsp

**problem**

clj-kondo reports unresolved-var for api usages where api is a namespace that uses import-var from potemkin.

**repro**

https://github.com/ericdallo/clojure-sample/blob/potemkin/src/clojure_sample/client.clj

First, lint the whole classpath from the project root with:
```bash
$ clj -Sdeps '{:deps {clj-kondo/clj-kondo {:mvn/version "2021.08.07-20210822.045523-16"}}}' -M -m clj-kondo.main --copy-configs --dependencies --lint $(clojure -Spath
```
then lint the api usage:

```bash
$ clj -Sdeps '{:deps {clj-kondo/clj-kondo {:mvn/version "2021.08.07-20210822.045523-16"}}}' -M -m clj-kondo.main --lint src/clojure_sample/client.clj
src/clojure_sample/client.clj:4:2: warning: Unresolved var: api/some-func                                                                                                                     
linting took 63ms, errors: 0, warnings: 1
```

**config**

No config.

**expected behavior**

Expect to not return `Unresolved var: api/some-func`

**Comment by borkdude** [comment]
The relevant code is in clj-kondo.impl.analyzer.potemkin
think it currently only supports potemkin syntax with some vector or so and not a single symbol
So this works:
```
(ns clojure-sample.api
  (:require
   [clojure-sample.impl.foo]
   [potemkin :refer [import-vars]]))

(import-vars
 [clojure-sample.impl.foo some-func])
 ```
